### ISSUE TITLE
I checked but I am not sure where is the DDL file ?

### ISSUE THREAD

**Issue by cpmangukiya** [oviradoi/npp-explorer-plugin#19]
I checked but I am not sure where is the DDL file ? #19

**Comment by oviradoi** [comment]
I'm not sure what you mean. Are you having trouble finding the DLL file of the explorer plugin?

If so, you can download the plugin from the Releases page of this github repo, and you need to unzip it to the Notepad++ plugin folder. Please check the Installation section in the readme file.

You can also install the plugin using Notepad++'s Plugin Admin. However, as of right now, the version you get from there is a bit older.

**Comment by cpmangukiya** [comment]
Hi,

For whatever stupid reasons I had no idea about this release page. I checked a lot and went here and there but could not find the DLL file or this ZIP file link. Now when you said I learned that there is "releases" page as well. Hope you give this "https://github.com/oviradoi/npp-explorer-plugin/releases" as hyper link in readme file to avoid confusion. There are other links in readme which distracted me more. Never mind. 

Thanks.
